src/components/MarkdownProvider/components/OverviewLinks.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { ISidebarSection } from '../../../layouts/Sidebar';
+import { LG, UnorderedList } from '@zendeskgarden/react-typography';
+import { Link } from '../../../layouts/Root/components/StyledNavigationLink';
+
+const StyledUnorderedList = styled(UnorderedList)`
+  margin: 0 0 ${p => p.theme.space.sm} 0;
+`;
+
+const StyledListItem = styled(UnorderedList.Item)`
+  list-style: none;
+`;
+
+export const OverviewLinks: React.FC<{ nav: ISidebarSection[] }> = ({ nav }) => {
+  const content = nav.map(section => (
+    <StyledUnorderedList key={`${section.title}`}>
```

Comment:
Since this string template doesn't involve additional string values this could be updated to not use string template. It would look like `key={section.title}`. Other string template values in this file could be updated in the same way.

Reply:
Good spot. I updated all the `key` instances.